docs/administration-guide/10-single-sign-on.md

```diff
@@ -40,39 +40,51 @@ Metabase will pull out three main attributes from your LDAP directory:
 
 - email (defaulting to the `mail` attribute)
 - first name (defaulting to the `givenName` attribute)
-- last name (defaulting to the `sn` attribute). 
-  
+- last name (defaulting to the `sn` attribute).
+
 If your LDAP setup uses other attributes for these, you can edit this under the "Attributes" portion of the form.
 
 ![Attributes](./images/ldap-attributes.png)
 
 Your LDAP directory must have the email field populated for each entry that will become a Metabase user, otherwise Metabase won't be able to create the account, nor will that person be able to log in. If either name field is missing, Metabase will use a default of "Unknown," and the person can change their name in their [account settings](../users-guide/account-settings.md).
 
+## LDAP advanced features
+
+Metabase Pro and Enterprise Editions ship with more advanced LDAP features.
+
+- [User schema](#ldap-user-schema)
+- [Syncing user attributes](#ldap-syncing-user-attributes)
+- [Group mapping](#ldap-group-mapping)
```

Comment:
Is this a change? When I was updating these docs last month, user schema and group mapping were open source features.